ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -264,6 +268,23 @@ const MachineSelector = ({onChange, selectedMachine, machineType, disabled, idPr
   </Fragment>;
 };
 
+const DisabledPanel = () => {
+  return <WarningMessage
+        iconSize={16}
+        iconStyle={{alignSelf: 'flex-start', flex: '0 0 auto'}}
```

Comment:
I'm not sure what this is doing differently from the default so my comment may not make sense but if its function is reusable in other contexts (ex. anchoring the icon to a different corner) it could be worth refactoring the component in such a way that clients pass in their desired "trait" and the component internalizes the styling.

Reply:
this is putting it in the top left corner instead of centered on the left. I can change it so it's top/middle/bottom for sure, can't imagine we'd want the alert on the right